packages/flutter/test/widgets/image_test.dart

```diff
@@ -1633,14 +1633,16 @@ void main() {
     ));
 
     // precacheImage is needed, or the image in the golden file will be empty.
-    final Finder allImages = find.byType(Image);
-    for (final Element e in allImages.evaluate()) {
-      await tester.runAsync(() async {
-        final Image image = e.widget as Image;
-        await precacheImage(image.image, e);
-      });
+    if (!kIsWeb) {
```

Comment:
Can you instead just skip the test for the browser with `skip: isBrowser` below? It should also have a comment referencing why the test is skipped so we remember to come back and turn it on when the issue is resolved.

Reply:
I do still want the most of this test to run on web to see how it behaves. The `precacheImage` is fixing the empty image issue probably caused by the non-web image cache (CC @dnfield for confirmation). Since I didn't see Skia gold to report any strange golden images on web, I think the web is fine with this test for now?

Reply:
Ok! Can we still reference an issue here for the `precacheImage` with a TODO so we know why this is is not happening on web and can enable it when it is fixed?

Reply:
Done.